src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,238 @@
+                    <img className='gr-7 gr-centered' src={it.url_for(`images/pages/mt5_signals/${image}.svg`)} />
+                </a>
+                <p>{text}</p>
+            </React.Fragment>
+            :
+            <React.Fragment>
+                <img className='gr-7 gr-centered' src={it.url_for(`images/pages/mt5_signals/${image}.svg`)} />
+                <p>{text}</p>
+            </React.Fragment>
+        }
+    </div>
+);
+
+const Box = ({ children }) => (
+    <div className='gr-12'>
+        <div className='gr-row full-height'>
+            {children}
+        </div>
+    </div>
+);
+
+const Mt5Signals = () => (
+    <div className='static_full'>
+        <div className='container'>
+            <h1 className='center-text'>{it.L('MT5 Signals')}</h1>
+            <TabContainer className='gr-padding-30 gr-parent full-width' theme='light'>
+                <TabsSubtabs
+                    id='mt5_signals_tabs'
+                    className='gr-parent tab-selector-wrapper'
+                    items={[
+                        { id: 'signal_subscriber', text: it.L('Signal Subscriber') },
+                        { id: 'signal_provider', text: it.L('Signal Provider') },
+                        { id: 'mt5_signals_tabs_selector', className: 'tab-selector' },
+                    ]}
+                />
+                <div className='tab-content'>
+                    <TabContentContainer>
+                        <TabContent id='signal_subscriber'>
+                            <div className='container gr-padding-20 gr-parent'>
+                                <p>{it.L('The MT5 trading signals service allows you to copy the trades of more experienced traders to your MT5 account. Once you’ve subscribed to a signal, the provider’s deals will be automatically replicated on your [_1] MT5 trading account each time they place a trade.', it.website_name)}
+                                </p>
+                            </div>
+                            <div className='fill-bg-color gr-padding-30 fullwidth' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('Benefits of subscribing to MT5 signals')}
+                                    </h2>
+                                    <div className='fill-bg-color'>
+                                        <div className='container section'>
+                                            <div className='gr-padding-10 facts'>
+                                                <div className='gr-row no-padding'>
+                                                    <Box>
+                                                        <BoxInner className='border-right-top' image='copy_trade' text={it.L('Minimise trading risk by copying from expert traders.', `<a href="${ it.url_for('group-history') }">`, '</a>')} />
```

Comment:
there are two anchor tags at the end of the text, but no placeholders in the text 🤔